server.js

```diff
@@ -224,132 +251,43 @@ io.on('connection', (socket) => {
-                        vf: votersForGov
-                    });
-                    votesForGov = 0;
-                    votesAgainstGov = 0;
-                    rejectedGovs = 0;
-                    votersAgainstGov = [];
-                    votersForGov = [];
-                }
-            }
+            checkVotes();
         });
 
         socket.on('no-for-gov', function (data) {
             votesAgainstGov++;
-            console.log("Players: " + shPlayers + "; Votes for: " + votesForGov + "; Votes against: " + votesAgainstGov + "; Total votes: " + (votesForGov + votesAgainstGov));
+//            console.log("Players: " + shPlayers + "; Votes for: " + votesForGov + "; Votes against: " + votesAgainstGov + "; Total votes: " + (votesForGov + votesAgainstGov));
             votersAgainstGov.push(data);
-
-            if (votesAgainstGov + votesForGov == shPlayers.length) {
-                console.log("all votes in");
-                if (votesAgainstGov >= votesForGov) {
-                    console.log("voting failed");
-                    nextPresident();
-                    io.emit('voting-failed', {
-                        presNom: presNom,
-                        pres: prevPresNom,
-                        chan: chanNom,
-                        va: votersAgainstGov,
-                        vf: votersForGov
-                    });
-                    votesForGov = 0;
-                    votesAgainstGov = 0;
-                    rejectedGovs++;
-                    votersAgainstGov = [];
-                    votersForGov = [];
-                    if (rejectedGovs >= 3) {
-                        undesirables = [];
-                        socket.emit('sh-chaos', {
-                            f: fPols,
-                            l: lPols
-                        })
-                        rejectedGovs = 0;
-                    }
-                } else if (votesForGov > votesAgainstGov) {
-                    console.log("voting passed");
-                    president = presNom;
-                    chancellor = chanNom;
-                    undesirables.push(president);
-                    undesirables.push(chancellor);
-                    io.emit('voting-passed', {
-                        pres: president,
-                        chan: chancellor,
-                        top: topThreePolicies,
-                        fPols: fPols,
-                        va: votersAgainstGov,
-                        vf: votersForGov
-                    });
-                    votesForGov = 0;
-                    votesAgainstGov = 0;
-                    rejectedGovs = 0;
-                    votersAgainstGov = [];
-                    votersForGov = [];
-                }
-            }
+            checkVotes();
         });
 
         socket.on('pres-chose-policies', function (data) {
             io.to('sh-chancellor').emit('policies-to-chancellor', data);
+            presPolicies = data;
+            botChan();
```

Comment:
stores policies in a variable to allow a bot to select from
bot picks policy if chancellor